phpdotnet/phd/Package/Generic/PDF.php

```diff
@@ -943,6 +949,13 @@ public function format_classsynopsisinfo_oointerface($open, $name, $attrs) {
 
     public function format_classsynopsis($open, $name, $attrs, $props) {
         if ($open) {
+            if (
+                isset($attrs[Reader::XMLNS_DOCBOOK]["class"]) &&
```

Comment:
I assume that the `class`-attribute is declared in the DTD. Otherwise it might be a good idea to rename that perhaps to "type" as that is describing better what we are referencing here.

Reply:
Yeah - this is correct. The class attribute is already defined with valid values of either class or interface.

https://tdg.docbook.org/tdg/4.5/classsynopsis.html

Reply:
More up to date version of the DTD we are aiming to use: https://tdg.docbook.org/tdg/5.1/classsynopsis.html

Reply:
Thanks for more current link @Girgias,  looks like it still supports the same attribute used in this PR tho. I just happened to relink to the original link @cmb69 provided the first time this method was suggested in a previous PR.